docs/application/native/guides/app-management/ui-app/efl-app.md

```diff
@@ -31,13 +36,16 @@ For more information on how to get started with EFL programming, see [Getting St
 With the EFL UI, you can create the following application types:
 
 - [Basic UI application](efl-ui-app.md)
-A basic UI application provides a DALi-based user interface which allows the user to interact with the application.The basic UI application is available for both mobile and wearable devices.
+
+  A basic UI application provides a DALi-based user interface which allows the user to interact with the application.The basic UI application is available for both mobile and wearable devices.
 
 - [Watch application](watch-app.md) **in wearable applications only**
```

Comment:
The link is broken. The `watch-app.md` file is under `/native/guides/app-management/`, not under `/native/guides/app-management/ui-app/`.

Reply:
done